lib/g10_certification_test_kit/bulk_data_group_export.rb

```diff
@@ -248,10 +248,12 @@ class BulkDataGroupExport < Inferno::TestGroup
         perform_export_kick_off_request
         assert_response_status(202)
 
-        polling_url = response[:headers].find { |header| header.name == 'content-location' }.value
+        polling_url = response[:headers].find { |header| header.name == 'content-location' }&.value
```

Comment:
You should instead use `request.response_header('content-location')&.value`, which handles case-insensitive matching of headers.